tests/foreman/cli/test_satellitesync.py

```diff
@@ -232,20 +381,30 @@ def class_export_entities(default_sat):
 
 
 @pytest.fixture(scope='function')
-def function_export_cv_directory(default_sat):
-    """Create directory for CV export and at the end remove it"""
-    export_dir = f'{get_export_base()}/{gen_string("alpha")}'
-    default_sat.execute(f'mkdir {export_dir}')
-    yield export_dir
+def export_cleanup_library(default_sat, function_org):
+    """Deletes dir for library export"""
+    yield
+    # Deletes directory created for library export test
+    assert default_sat.execute(f'rm -rf {EXPORT_DIR}/{function_org.name}').stdout == ''
+
+
+@pytest.fixture(scope='function')
+def export_cleanup(default_sat, module_org):
+    """Deletes dir for version export"""
+    yield
     # Deletes directory created for CV export Test
-    default_sat.execute(f'rm -rf {export_dir}')
+    assert default_sat.execute(f'rm -rf {EXPORT_DIR}/{module_org.name}').stdout == ''
 
 
-def get_export_base():
-    return '/var/lib/pulp/katello-export'
+def validate_filepath(default_sat, EXPORT_DIR):
```

Comment:
I don't think you mean to pass `EXPORT_DIR` here, since its now defined at the module level.